src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    public static final String SPLITTER_REGEX = "\\W+";
+    public static final String BUY_STRING_VALUE = "buy";
+    public static final String SUPPLY_STRING_VALUE = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method name 'getStatistic' could be more descriptive of its function. According to the checklist, method names should be informative. Consider renaming it to something like 'generateReport' to better reflect its purpose.